anet.yml

```diff
@@ -330,3 +330,9 @@ dictionary:
         options:
           layers: GEBCO_LATEST
           format: "image/png"
+
+  automaticallyInactivateUsers:
+    emailReminders:
+      daysPrior: [15, 30, 45]
+    clearDomainUserName:
```

Comment:
The actual (database) property is called `domainUsername` without the capital `N` in _Name_.

Reply:
Yes, but since I will be renaming it to 'ignoredDomainNames' this no longer applies. I will keep it in mind though...